teuthology/orchestra/remote.py

```diff
@@ -53,6 +55,9 @@ def __init__(self, name, ssh=None, shortname=None, console=None,
         self._console = console
         self.ssh = ssh
 
+        if self._reimage_types is None:
+            self._reimage_types = teuthology.provision.get_reimage_types()
+
```

Comment:
If you wished to set static class parameter I think you need to use `Remote._reimage_types` instead of `self._reimage_types`.

Reply:
For the code:
```
class A(object):
    _x = None
    def __init__(self, name):
        if self._x is None:
            print(f"Setting _x for {name}")
            self._x = "A"

class B(object):
    _x = None
    def __init__(self, name):
        if B._x is None:
            print(f"Setting _x for {name}")
            B._x = "B"

a1 = A('a1')
a2 = A('a2')
print(f'a1={a1._x}')
print(f'a2={a2._x}')

b1 = B('b1')
b2 = B('b2')
print(f'b1={b1._x}')
print(f'b2={b2._x}')
```

we have the output:

```
Setting _x for a1
Setting _x for a2
a1=A
a2=A
Setting _x for b1
b1=B
b2=B
```

Reply:
ACK, done